### ISSUE TITLE
Allow switching accounts from the post composer

### ISSUE THREAD

**Issue by gesteves** [bluesky-social/social-app#5531]
**Is your feature request related to a problem? Please describe.**

Sometimes I start writing a post and then realize I'm logged into the wrong account. This can be particularly annoying if it's a reply, since then I have to exit the composer, switch accounts, find the post I wanted to reply to, and then re-type my post. It'd be nice to be able to quickly switch accounts from the post composer without losing what I've typed so far.

**Describe the solution you'd like**

Perhaps tapping my avatar in the composer could open the account switcher so I could quickly select which account to post from.

**Describe alternatives you've considered**

1. Copy the post I wrote to the clipboard
2. Switch accounts
3. If I was replying to someone, go find the post I wanted to reply to
4. Paste the text I had written (or write it from scratch)
5. Publish the post

**Additional context**

N/A

**Comment by scottdotweb** [comment]
I'd like to see account switching available from the main menu as well (basically, similar to Twitter) rather than needing to dive into the settings page. Perhaps this request could be broadened?

**Comment by surfdude29** [comment]
> I'd like to see account switching available from the main menu as well (basically, similar to Twitter) rather than needing to dive into the settings page. Perhaps this request could be broadened?

On mobile, you can long-press on your avatar to switch accounts, in case that's helpful.

**Comment by scottdotweb** [comment]
Oh! It is, thank you. There's no obvious reason to long-press it (at least, to me) so I would never have found that out myself.

**Comment by surfdude29** [comment]
There were some suggestions and discussions about how to improve the discoverability of the quick account switcher in #2800 some months ago; it should probably be revisited.

**Comment by surfdude29** [comment]
Also FYI the quick account switcher is being updated in #5881 to add the ability to remove an account and log out from the current account.

**Comment by scottdotweb** [comment]
Love to see it. Thanks for the info!

**Comment by halans** [comment]
Regardless of the current quick account switcher, 
I would also love to see an account switcher on the post composer, as I don't necessarily want to switch account context.

**Comment by scottdotweb** [comment]
Absolutely, very useful for quickly reposting something on a different account without disturbing your current reading flow.

**Comment by shredderik** [comment]
Deck.blue allows for selecting the account from which to post or quote/repost by displaying the different account avatars in the post composer and allowing for an easy switch by tapping the respective avatar. Having this is the main bsky android client would be a godsend.

**Comment by rvyhvn** [comment]
The switch account on long press doesn't work.
